packages/core/src/router.ts

```diff
@@ -31,6 +31,7 @@ import {
 } from './types'
 import { hrefToUrl, mergeDataIntoQueryString, urlWithoutHash } from './url'
 
+const isChromeIOS = /CriOS/.test(navigator.userAgent)
```

Comment:
Have you tested this in SSR by any chance to make sure it works? I'm worried it might blow up because `navigator` isn't available. Also, should it be `window.navigator`?

Reply:
You are right, I'll fix this and test once again with SSR.

Reply:
Fixed ✅